### PR TITLE
rgw/test: Changing force-branch to master

### PR DESCRIPTION
**Pull request by TRYTOBE8TME**
This PR includes changing force-branch to master and removing the git-remote. This change was forgetten for PR #39139.

Signed-off-by: Kalpesh Pandya <kapandya@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by yuvalif — APPROVED** [review]

**Comment by yuvalif** [comment]
please add a link to teuthology run

**Comment by cbodley** [comment]
thanks @TRYTOBE8TME!

in our other test repos (s3-tests etc), we use different branches for each ceph release, and name them ceph-master, ceph-pacific, ceph-octopus, etc. that way, we can make sure that when we're testing ceph's pacific release, we only run the test cases that are supported on pacific

it also helps to keep 'master' separate from 'ceph-master', so we can develop on master and merge tests there before the related ceph functionality is merged to ceph's master branch

so can you please create a ceph-master branch on that repo, and target that in this pr?

**Comment by cbodley** [comment]
> (irc) try_to_be8t_me: But the tests reside in the ceph repo itself

hmm, i guess master is ok

ideally we'd either default to the suite_branch we're running against, or just copy the test-specific files out of teuthology's suite_path onto the worker node so we don't have to clone the entire ceph tree from github every time

**Comment by TRYTOBE8TME** [comment]
Please do not merge now as I'm testing this in teuthology.

**Comment by TRYTOBE8TME** [comment]
Teuthology clean run: https://pulpito.ceph.com/kapandya-2021-04-07_15:43:41-rgw:notifications-wip-rgw-notifications-distro-basic-smithi/

File: qa/tasks/kafka.py
Comment (yuvalif) [inline qa/tasks/kafka.py:115]: are you sure this is needed?
isn't the change in line 58 enough (together with proper shutdown)?

Comment (TRYTOBE8TME) [reply qa/tasks/kafka.py:115]: The change at line 58 is removing the kafka-2.6.0-src/logs. So, probably this is required.

Comment (yuvalif) [reply qa/tasks/kafka.py:115]: the change in line 58 is required.
however, I'm not sure we should use `kill -9` here. probably better to use the proper "close" scripts

Comment (TRYTOBE8TME) [reply qa/tasks/kafka.py:115]: The close scripts were giving some failures (as discussed), so I used this option.

Comment (yuvalif) [reply qa/tasks/kafka.py:115]: did you try the combination of the close scripts and the deletion of the logs directory?

Comment (TRYTOBE8TME) [reply qa/tasks/kafka.py:130]: I've scheduled a run for that.

Comment (TRYTOBE8TME) [reply qa/tasks/kafka.py]: The combination of close scripts and logs directory deletion worked now: https://pulpito.ceph.com/kapandya-2021-04-07_19:03:41-rgw:notifications-wip-rgw-notifications-distro-basic-smithi/

**Pull request merged**
